Scripts/application.py

```diff
@@ -7,49 +7,49 @@
 app = Flask(__name__)
 
 path_to_db = '../db/allYourSantaAreBelongToUs.db'
+BAD_REQUEST_ERROR = 400
 
 @app.route('/')
 def hello_world():
     return 'Hello World!'
     
 @app.route('/api/v1/person')
 def get_all_people():

```

Comment:
get_all_people -> get_person

Reply:
ok

Reply:
Well this one has to be get_persons() to distinguish it from the other get_person() method.